lib/manager/npm/extract/index.ts

```diff
@@ -213,6 +213,9 @@ export async function extractPackageFile(
       } else if (depName === 'yarn') {
         dep.datasource = npmId;
         dep.commitMessageTopic = 'Yarn';
+      } else if (depName === 'npm') {
+        dep.datasource = npmId;
+        dep.commitMessageTopic = 'npm';
```

Comment:
```suggestion
```

There is no need to specify the `commitMessageTopic` if it's the same as `depName`